p2p/peer.py

```diff
@@ -334,6 +348,8 @@ def is_closing(self) -> bool:
         # though, otherwise asyncio's event loop can't run and we can't keep up with other peers.
         decoded_msg = cast(Dict[str, Any], cmd.decode(msg))
         self.logger.trace("Successfully decoded %s msg: %s", cmd, decoded_msg)
+        count = self.received_msgs.get(cmd, 0)
+        self.received_msgs[cmd] = count + 1
```

Comment:
If you make `received_msgs` a `collections.defaultdict(int)` then you can make this a one-liner as `self.received_msgs[cmd] += 1`